sematic/ui/src/utils.tsx

```diff
@@ -67,6 +68,18 @@ export function useLogger() {
   }
 } 
 
+
+export function atomWithHashCustomSerialization(
+  name: string, initialValue: string, 
+  options: Parameters<typeof atomWithHash>[2] = {}) {
+  let overridenOptions = options || {};
+  // Use custom serialization function to avoid generating `"`(%22) in the hash
+  overridenOptions.serialize = (value: unknown) => (value as any).toString() ;
+  overridenOptions.deserialize = (value: unknown) => value as string ;
+
+  return atomWithHash<string>(name, initialValue, options as any); 
```

Comment:
The reason why we have to cast to `any` is because on line 74, we can't do `Parameters<typeof atomWithHash<string>>[2]` This syntax is supported in TypeScript 4.7.0. (we are at 4.6.4)